TESTING-INSTRUCTIONS.md

```diff
@@ -2,6 +2,24 @@
 
 ## Unreleased
 
+### Store profiler - Added MailPoet to new Business Details step  #6515
+
+- Create a brand new site and go to the OBW.
+- In the first OBW step (`Store Details`) set `US` in the `Country / Region` selector.
+- Set `Food and drink` in the second step (`Industry`)
```

Comment:
Given https://github.com/woocommerce/woocommerce-admin/pull/6508 is merged, this is not required anymore, as long as it is not the CBD option. Country list is expanded now as well, as mentioned in the other PR, but sticking with `US` here is fine.

Reply:
Sounds good to me 🙂
This was changed in the commit b43b1a0